docs/docs/monitoring.md

```diff
@@ -86,6 +93,24 @@ When Prometheus is enabled in the status plugin (see [Configuration](./configura
 | last_success_bundle_request    | gauge       | Last successful bundle request in UNIX nanoseconds.    | STABLE |
 | bundle_loading_duration_ns     | histogram   | A histogram of duration for bundle loading.            | STABLE |
 
+## Available Metrics
+
+The Prometheus `/metrics` endpoint exposes the following instance-level metrics:
+
+- **URL**: `http://localhost:8181/metrics` (default configuration)
+- **Method**: HTTP GET
+- **Format**: Prometheus text format
+- **Contents**: Instance-level counters, timers, histograms, Go runtime metrics
+- **Use case**: Monitoring dashboards, alerting, performance trends
+
```

Comment:
Please list 

- http_request_duration_seconds - Request latency histogram by endpoint, method, and status code
- then link to https://pkg.go.dev/runtime/metrics#hdr-Supported_metrics for the go ones and leave them to document that